x-pack/plugins/ml/public/settings/scheduled_events/components/new_event_modal/new_event_modal_controller.js

```diff
@@ -42,7 +42,11 @@ module.controller('MlNewEventModal', function (
       start: moment().startOf('day'),
       end: moment().startOf('day').add(1, 'days')
     },
-    setRangeType(i) {
+    setRangeType(i, event = undefined) {
```

Comment:
nit, i believe explicitly setting `event` to `undefined` here is redundant.
If nothing is passed in, it'll be `undefined` anyway? unless i'm missing something?

Reply:
Not missing anything - just old habits. I'm good with removing it 👌